buildbot-conf/private.py.sample

```diff
@@ -0,0 +1,11 @@
+pbcsUser    = 'password'
+pbcsPwd     = 'password'
+webAuthUser = 'password'
+webAuthPwd  = 'password'
+
+slave34Pwd = 'password'
```

Comment:
Please use `pass` for backward compatibility. I.e. current configuration uses `pass` as a slave password.